.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+# Set update schedule for GitHub Actions
+
+version: 2
+updates:
+  - package-ecosystem: "nuget"
+    directory: "/"
```

Comment:
Can you use the specific directory? See reference: https://github.com/oras-project/oras-dotnet/blob/main/.github/dependabot.yml

Reply:
Updated.